ios/FluentUI/Tooltip/TooltipViewController.swift

```diff
@@ -9,27 +9,29 @@ import UIKit
 
 class TooltipViewController: UIViewController {
 
-    init(anchorView: UIView,
+    init(anchorView: @escaping () -> UIView?,
          message: String,
          title: String? = nil,
          textAlignment: NSTextAlignment,
          preferredArrowDirection: Tooltip.ArrowDirection,
          offset: CGPoint,
-         arrowMargin: CGFloat,
-         tokenSet: TooltipTokenSet) {
+         arrowMargin: CGFloat) {
+        guard let anchorView = anchorView() else {
```

Comment:
Could you talk through the advantages of using a callback here instead of just passing the anchorView directly through?

Reply:
I was using your closure slides from the PPT Harrie sent as reference when doing this and thought it was needed to keep anchorView weak! But if that's not the case, I can change it back to just passing anchorView